R/board_rsconnect.R

```diff
@@ -324,6 +324,12 @@ pin_search.pins_board_rsconnect <- function(board, search = NULL, ...) {
   multi_meta(board, paste0(user, "/", name))
 }
 
+
+#' @export
+board_deparse.pins_board_rsconnect <- function(board, ...) {
+  glue('board_rsconnect(server = "{board$server}")')
```

Comment:
Honestly a little unsure if this should be `server` or `server_name`

Reply:
We'll probably need to think this through carefully. I suspect it depends on exactly which authentication you used 😬

Reply:
OMG `board$server` only works in my particular situation because of partial matching 💀 

```r
> names(b)
 [1] "board"                "api"                  "cache"                "name"                
 [5] "url"                  "account"              "server_name"          "auth"                
 [9] "versioned"            "use_cache_on_failure"
```

Reply:
Take a look at how I approached this; I think it will work for the various auth strategies there are for RSC.